### PR TITLE
ADABR

### PR DESCRIPTION
**Pull request by tauanbinato**
Adding ADABR proof of donations.

### CODE REVIEW

File: ADABR/donations.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    contact: chalanaesperanca@espacosilvestre.org.br
+    website: https://www.instagram.com/chalanaesperanca/
+  ada: 519.91
+  usd: 965.17
+  timestamp: 1629342000
+  files:
+     - ADABR/01_ce.png
+     
+- ticker: ADABR
+  announcement_url: https://www.instagram.com/p/CSwwplPNZ7z/
```
Comment (ResponsibleStaking) [inline ADABR/donations.yml:50]: This is the same announcement_url as above. Propably a copy paste issue.

**Review by ResponsibleStaking — CHANGES_REQUESTED** [review]

**Comment by tauanbinato** [comment]
> Hi @tauanbinato ! Please give feedback to the above comment. I'd like to clean out old PRs.

Done, thanks

**Comment by nova4005** [comment]
Hey, I was checking this and the automated checks are failing for no 'tel' on some of your recipients. If you can add those in I can continue with the review.

**Comment by nova4005** [comment]
Closing this since we have not received a response. You can reopen once the updates are completed.

**Pull request closed without merging**